### PR TITLE
Update AUR documentation since the package got moved to Arch Extra

### LINKED ISSUE #2303: AUR package from the docs doesn't exist anymore
### Bug report info

```plain text
Issue exists before installing.
```

### Command used with act

```sh
Issue exists before installing.
```

### Describe issue

The AUR package `act` that is linked in the docs, doesn't seem to exist anymore.
There is `act-bin` (which is prebuilt) and `act-git` (which is completely out-of-date), but no `act`.
Why was it removed?

### Link to GitHub repository

_No response_

### Workflow content

```yml
Not relevant
```

### Relevant log output

```sh
Not relevant
```

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by MatthiasKunnen**
Arch upgraded `act` from the AUR to its "Extra" repository. This PR changes the documentation accordingly.

This is an alternative to #20. The reason for this is that, in my opinion, _Arch_ is the better name over _pacman_ for these reasons:
- The package is in the [Arch Extra](https://archlinux.org/packages/extra/x86_64/act/) repository not in [Manjaro Extra](https://gitlab.manjaro.org/packages/extra) or any other.
- Pacman is a package manager, not a repository. The installation sections starts with

  > … below package repositories:

  In addition, pacman could be pointed to non-arch repos that do not include `act`.
- I believe the usage of the term _Arch Linux_ is more common than _Pacman_ when installation instructions are provided.

### CODE REVIEW

File: src/SUMMARY.md
Comment (MatthiasKunnen) [inline src/SUMMARY.md]: I could also name it `Arch Linux`, let me know and I'll change it and other mentions.

Comment (ChristopherHX) [reply src/SUMMARY.md]: missing .md extension? idk if mdbook doesn't care, but for consistency

Otherwise I would give this pr my approval.

Comment (MatthiasKunnen) [reply src/SUMMARY.md]: Good catch, fixed it

File: src/installation/arch.md
Comment (ChristopherHX) [inline src/installation/arch.md]: Per https://github.com/nektos/act/pull/1355, we should not suggest dangerous pacman commands.

```suggestion
pacman -Syu act
```

**Comment by ChristopherHX** [comment]
For me it doesn't matter if if you squash or not, because I would do that in GitHub UI anyways.

Not shure how cplee handles manual merges as I don't have that much power here to do that myself.

File: src/installation/arch.md
Comment (MatthiasKunnen) [reply src/installation/arch.md]: hmm, this is a bit strange to me.

`pacman -S act` is not dangerous by itself, it is only dangerous if it preceded by `pacman -Sy`. The `Syu` combo performs a full system upgrade + installs act.

Is the thinking behind using `Syu` to protect a user that has run `pacman -Sy` previously without knowledge of the implications and then copy pastes `pacman -S act` from the documentation?

If that is the case, I understand. Experienced Arch users can always remove the `yu`.

**Review by gobborg — APPROVED** [review]

File: src/installation/arch.md
Comment (MatthiasKunnen) [reply src/installation/arch.md]: @ChristopherHX I've applied your suggestion

**Review by ChristopherHX — APPROVED** [review]

File: src/installation/arch.md
Comment (ChristopherHX) [reply src/installation/arch.md]: Tbh, I'm no longer that active under nektos/act ( I'm still the most active maintainer who writes code ) and don't think I'm in the position to revert contributions that were explicitly reviewed by the owner cplee (who controls this GitHub Org and nektosact.com as the only one I'm aware of)

> Is the thinking behind using Syu to protect a user that has run pacman -Sy previously without knowledge of the implications and then copy pastes pacman -S act from the documentation?

Yes this is at least my reason, because nektosact has a potential big audience.

You are probably right that the pacman command without full system upgrade flag is not that dangerous (compared to the old yay command, that depends on go), since act has no dependencies except docker (I don't know how far the system integrity can shift if this is implicitly updated / installed).

Thanks for applying my change, you have my approval for shure now.

My preference at this point of time, don't make a lot of unrelated changes when making changes like this.
So there are much less topics to discuss, not saying that an follow up couldn't be considered.

**Review by catthehacker — APPROVED** [review]
Generally would prefer it to be called `ArchLinux` rather than just `Arch`, but there has been enough bikeshedding about whole thing

**Pull request merged**
